### PR TITLE
Crypto Badges added `new-badge`

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
New badges added `Z Cash` `Tether` `Polkadot` and `Binance`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,15 +154,19 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
...
 | Iota         | ![Iota](https://img.shields.io/badge/iota-29334C?style=for-the-badge&logo=iota&logoColor=white)                             | `![Iota](https://img.shields.io/badge/iota-29334C?style=for-the-badge&logo=iota&logoColor=white)`                             |
 | Litecoin     | ![Litecoin](https://img.shields.io/badge/Litecoin-A6A9AA?style=for-the-badge&logo=Litecoin&logoColor=white)                 | `![Litecoin](https://img.shields.io/badge/Litecoin-A6A9AA?style=for-the-badge&logo=Litecoin&logoColor=white)`                 |
 | Monero       | ![Monero](https://img.shields.io/badge/monero-FF6600?style=for-the-badge&logo=monero&logoColor=white)                       | `![Monero](https://img.shields.io/badge/monero-FF6600?style=for-the-badge&logo=monero&logoColor=white)`                       |
+| Polkadot     | ![Polkadot](https://img.shields.io/badge/polkadot-E6007A?style=for-the-badge&logo=polkadot&logoColor=000)                   | `![Polkadot](https://img.shields.io/badge/polkadot-E6007A?style=for-the-badge&logo=polkadot&logoColor=000)`                    |
```
Comment (Ileriayo) [inline README.md:165]: Any reason in particular why you used `logoColor=000` unlike the general pattern?

File: README.md [hunk 2]
```diff
@@ -154,15 +154,19 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
...
 | Stellar      | ![Stellar](https://img.shields.io/badge/Stellar-7D00FF?style=for-the-badge&logo=Stellar&logoColor=white)                    | `![Stellar](https://img.shields.io/badge/Stellar-7D00FF?style=for-the-badge&logo=Stellar&logoColor=white)`                    |
+| Tether       | ![Tether](https://img.shields.io/badge/tether-168363?style=for-the-badge&logo=tether&logoColor=white)                       | `![Tether](https://img.shields.io/badge/tether-168363?style=for-the-badge&logo=tether&logoColor=white)`                      |
 | Xrp          | ![Xrp](https://img.shields.io/badge/Xrp-black?style=for-the-badge&logo=xrp&logoColor=white)                                 | `![Xrp](https://img.shields.io/badge/Xrp-black?style=for-the-badge&logo=xrp&logoColor=white)`                                 |
+| Z Cash       | ![Z Cash](https://img.shields.io/badge/Zcash-F4B728?style=for-the-badge&logo=zcash&logoColor=000)                           | `![Z Cash](https://img.shields.io/badge/Zcash-F4B728?style=for-the-badge&logo=zcash&logoColor=000)`                          |
```
Comment (Ileriayo) [inline README.md:169]: Here too

File: README.md (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply README.md:165]: Yeah because If I use white then badge icons not show correctly

**Review by Ileriayo — APPROVED** [review]
LGTM

**Pull request merged**